VideoApi/Testing.Common/Helper/Builders/Api/BookNewConferenceRequestBuilder.cs

```diff
@@ -31,13 +31,24 @@ public BookNewConferenceRequestBuilder(string caseName)
 
         public BookNewConferenceRequestBuilder WithJudge(string firstName = null)
         {
+            return WithJudicialParticipant(UserRole.Judge, firstName);
+        }
+        
+        public BookNewConferenceRequestBuilder WithJudicialOfficeHolder(string firstName = null)
+        {
+            return WithJudicialParticipant(UserRole.JudicialOfficeHolder, firstName);
+        }
+
+        private BookNewConferenceRequestBuilder WithJudicialParticipant(UserRole role, string firstName = null)
+        {
+            var hearingRole = role == UserRole.Judge ? "Judge" : "PanelMember";
```

Comment:
Could this also be an enum?

Reply:
A hearing role is a string value across all the APIs because the possible hearing roles are based on the case and hearing type.

Reply:
Please look over previous sessions for any reminders